server/views/partials/showLocation.njk

```diff
@@ -0,0 +1,30 @@
+{% macro showLocation(event, internalLocation, makeCapitals = false) %}
```

Comment:
Is `makeCapitals` for use later?  Unless I am mistaken I can't see it being used, always false.